configure.py

```diff
@@ -1369,14 +1369,14 @@ def innosetup_arch(os, arch):
     if options.os != 'windows':
         vars['botan_pkgconfig'] = prefix_with_build_dir(os.path.join(build_config.build_dir,
                                                                      build_config.pkg_config_file()))
-
-    vars["header_in"] = process_template('src/build-data/makefile/header.in', vars)
+    proj_dir = os.path.dirname(os.path.realpath(__file__)) + '/'
```

Comment:
Relying on directories to have a trailing "/" is most of the time not a good idea. It's safer and more consistent when you add the slash where you use it.